torch/utils/_sympy/reference.py

```diff
@@ -421,7 +421,7 @@ def int_truediv(a, b):
 
     @staticmethod
     def floordiv(a, b):
-        return torch.ops.aten.floor_divide(a, b)
+        return torch.ops.aten.div.Tensor_mode(a, b, rounding_mode = 'floor')
```

Comment:
I couldn't get `return torch.ops.aten.floor_divide.default(a, b)` working for the `python test/inductor/test_torchinductor_dynamic_shapes.py TestInductorDynamicCUDA.test_interpolate_ceil_eq_cuda` test.

It would output

```
MissingOperatorWithDecomp: missing decomposition
  target: aten.floor_divide.default
  args[0]: TensorBox(StorageBox(
    Pointwise(
      'cpu',
      torch.float32,
      def inner_fn(index):
          tmp0 = ops.constant(-1, torch.int64)
          tmp1 = ops.to_dtype(tmp0, torch.float32, src_dtype=torch.int64)
          tmp2 = ops.index_expr(s1, torch.float32)
          tmp3 = tmp1 + tmp2
          return tmp3
      ,
      ranges=[],
      origin_node=add_tensor,
      origins=OrderedSet([add_tensor, scalar_tensor_default_3, full...
    )
  ))
  args[1]: TensorBox(StorageBox(
    Pointwise(
      'cpu',
      torch.int64,
      def inner_fn(index):
          tmp0 = ops.constant(8, torch.int64)
          return tmp0
      ,
      ranges=[],
      origin_node=full_default_3,
      origins=OrderedSet([full_default_3])
    )
  ))

There is a decomposition available for aten.floor_divide.default in
torch._decomp.get_decompositions().  Please add this operator to the
`decompositions` list in torch._inductor.decomposition
```

But adding `aten.floor_divide.default` to the list didn't fix it. If reviewers have any suggestions on how to get `torch.ops.aten.floor_divide` to work properly, I'm happy to make the change.

Reply:
No, your change here looks right, thanks